Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision b00e35ed5c055d141bc1e987788f5c95cb8085d9)
+++ .gitignore	(date 1626615780470)
@@ -33,4 +33,4 @@
 .vscode/
 
 ## MY ADDITION ###
-.idea/workspace.xml
+#.idea/workspace.xml
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/.gitignore	(revision b00e35ed5c055d141bc1e987788f5c95cb8085d9)
+++ .idea/.gitignore	(date 1626615780486)
@@ -1,6 +1,6 @@
 # Default ignored files
 /shelf/
-/workspace.xml
+#/workspace.xml
 # Datasource local storage ignored files
 /dataSources/
 /dataSources.local.xml
